organization/hsf-letters.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Letters from the HSF"
+author: "Graeme Stewart"
+layout: default
+---
+
+# {{page.title}}
+
+The HSF's mission is to encourage cooperation and common projects
+between all parts of the high-energy physics community concerned
+with software. We span regions and experiments and we strive to 
+to make our community's software development more effective.
+To that end, the HSF will happily cooperate with any project that
+shares similar goals. 
+
+### Letters of Cooperation
+
+We will be happy to provide a *letter of cooperation*
+that can be used for any funding application. To ask us
+for this please:
+
+- Send an email request to [HSF Coordination](mailto:hsf-coordination@googlegroups.com)
+- Include an outline of your project's goals and indicate where you
+  believe that the HSF can usefully cooperate with you
+- Please make this request at least *two weeks* before you need the letter
+
+### Letters of Support
+
+If you would like a *letter of support* for your project then you
+should additionally show how your project will help to meet the goals
+of the HSF as set out, for example, in the [Community White Paper](/organization/cwp.html).
+Please note that a letter of support will not be exclusive for any
+particular grant application round - the HSF is free to support multiple
+projects.
+
+- Send an email request to [HSF Coordination](mailto:hsf-coordination@googlegroups.com)
+- Include an outline of your project's goals and indicate in what way
+  this helps the high-energy physics community to meet the software challenges
+  of the future
+- Please make this request at least *three weeks* before you need the letter
+
+### Procedure
+
+In both cases the HSF Coordination group will discuss the request and, if approved,
+circulate a draft letter to the whole of the HSF community for any comments. We will
+then approve the request in an HSF Coordination meeting (or, exceptionally, just by email,
+should deadlines not fit with the meeting schedule).
```

Comment:
Is not this too much an implementation detail? Why not something like "The request will be discussed and eventually/hopefully approved by the HSF Coordination"?

Reply:
Well, I think it's an important detail, that their request will be publically discussed. We can always update it later if we change this.

Reply:
I agree that the public flavour of the approval is important since the HSF is about the community. It prevents any feeling of exclusion or bias for what should be "global decisions".